searchcore/src/vespa/searchcore/proton/server/feedhandler.h

```diff
@@ -73,9 +75,11 @@ class FeedHandler: private search::transactionlog::client::Callback,
     std::unique_ptr<TlsWriter>             _tlsMgrWriter;
     TlsWriter                             *_tlsWriter;
     TlsReplayProgress::UP                  _tlsReplayProgress;
-    // the serial num of the last message in the transaction log
+    // the serial num of the last feed operation processed by feed handler.
     SerialNum                              _serialNum;
     SerialNum                              _prunedSerialNum;
+    // the serial num of the last feed operation in the transaction log at startup before replay
```

Comment:
Consider adding a comment to __prunedSerialNum_ as well.

Reply:
Done.